solvable_files.sh

```diff
@@ -118,6 +121,10 @@ function correct_mtime() {
 	if [ "$action" = "fix" ]
 	then
 		touch --no-create "$filepath"
+		if [ "$scan_action" = "scan" ]
+		then
+			sudo -u www-data php ./occ files:scan --quiet --path="${relative_filepath}"
```

Comment:
This introduces requirements on how to run occ, having sudo, using the www-data user and being in the nextcloud root directory. Let's go got that now, if someone needs more, we'll introduce a better way of passing arguments to let the user have more control over it.

```suggestion
			sudo -u www-data php ./occ files:scan --quiet --path="$relative_filepath"
```

Reply:
yes sorry forgot about that because I was in a hurry

Reply:
Still need that one.

Reply:
yes sorry just saw it after

Reply:
@artonge  done, sorry